UKHomeOffice/sgmr-service
# SPCR-378 Use country selector autocomplete for pc registration country

### AC / Description of changes
- update the pleasure craft form to use the autocomplete for country of registration
- small refactor of the country selector combobox field to make it usable for this form as well as others

### To Test

- go to the pleasure craft page
- save a new pleasure craft
- fill out page one and continue
- on page 2 select YES to country of registration
- start typing in enter country of registration field
- > you should see suggested countries based on your input
- > you should always see British Citizen at the top of the list


### Notes

The RefData service defines the country names that are used in this list, that defines 'British Citizen' rather than 'United Kingdom'. Changes to this list go via the data teams

---
* remember to merge to feature branches not master *
